src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -31,29 +31,29 @@ public static void M1()
 }
 ";
             string expectedOperationTree = @"
-IUsingStatement (OperationKind.UsingStatement) (Syntax: 'using (var  ... }')
+IUsingOperation (OperationKind.Using, IsStatement, Type: null) (Syntax: 'using (var  ... }')
   Resources: 
-    IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'var c = new C()')
-      IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'c = new C()')
+    IVariableDeclarationsOperation (1 declarations) (OperationKind.VariableDeclarations, IsStatement, Type: null) (Syntax: 'var c = new C()')
```

Comment:
>IVariableDeclarationsOperation (1 declarations) (OperationKind.VariableDeclarations, IsStatement, Type: null) (Syntax: 'var c = new C()') [](start = 4, length = 137)

Why is this a statement? Feels unexpected.

Reply:
Syntactically this isn't a statement. Before we simply used a node that had "Statement" in its name because that was a good fit and it wasn't worth introducing a separate node for this situation.